src/orchestrator/orchestrator.rs

```diff
@@ -562,15 +567,18 @@ impl<'a> Drop for JobTask<'a> {
             let errmsg = if self.jobdef.dependencies.is_empty() {
                 "error occurred"
             } else {
-                "error on other task"
+                "inherited"
```

Comment:
Hm, "inherited" seems a bit brief - "error (inherited)" would seem better but I guess we want to avoid the term error for "filtering"?

We should also be able to propagate the dependency/task where the error occurred (via `self.jobdef.dependencies`). This could be quite useful but I guess it's not necessary (and we'd have to decide if we output all dependencies where errors occurred or just the first one).

Reply:
I was told to name if "inherited"

Reply:
@christophprokop was this your suggestion and is "inherited" clear enough?
I need to have a closer look but maybe we could also use something like "aborted" or "skipped" (based on the diff it looks like this case is for jobs that failed due to a propagated error from their dependencies)?